samples/js/splore/src/main.js

```diff
@@ -222,30 +248,38 @@ function handleNodeClick(e: MouseEvent, id: string) {
   }
 }
 
-class Prompt extends React.Component<void, {}, void> {
+type PromptProps = {
+  msg: string,
+}
+
+class Prompt extends React.Component<void, PromptProps, void> {
   render(): React.Element<any> {
     const fontStyle: {[key: string]: any} = {
       fontFamily: 'Menlo',
       fontSize: 14,
     };
     const inputStyle = Object.assign(fontStyle, {}, {width: '50ex', marginBottom: '0.5em'});
+    const demoServer = 'https://demo.noms.io/cli-tour';
+
     return <div style={{display: 'flex', height: '100%', alignItems: 'center',
       justifyContent: 'center'}}>
       <div style={fontStyle}>
-        Can haz database?
+        {this.props.msg}
         <form style={{margin:'0.5em 0'}} onSubmit={e => this._handleOnSubmit(e)}>
           <input type='text' ref='db' autoFocus={true} style={inputStyle}
-            defaultValue={params.db || 'https://demo.noms.io/cli-tour'}
-            placeholder='noms database URL'
+            defaultValue={params.db}
+            placeholder={`database URL (${demoServer})`}
+          />
+          or
+          <input type='text' ref='p' style={inputStyle}
```

Comment:
I guess that is different because in the case of `show` you can only provide a path. Providing a db is meaningless. In this case, I think I would prefer to have the path field just be the path alone.